### PR TITLE
completes step 8 - refactored tests to make them more consistent

### PR DESCRIPTION
**Pull request by jcainuk**
Completes PATCH for step 8

### CODE REVIEW

File: app.test.js [hunk 1]
```diff
@@ -125,6 +127,72 @@ describe("/api/articles/:article_id", () => {
...
+    });
+    test("400: responds with error when no votes are are included", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({})
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toBe("Bad request");
+        });
+    });
```
Comment (johnhutcheon) [inline app.test.js:194]: Great testing so far! Could you just add one more please? What do we think might happen if we're sending inc_votes and it's not a number? I.e. {inc_votes: "banana"} ?

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Hey Jonathan, really good work so far! Could you just add the further test as per my comment and resubmit please? Thanks! 🙂

File: app.test.js [hunk 2]
```diff
@@ -192,6 +192,24 @@ describe("/api/articles/:article_id", () => {
...
+    test("POST 400: responds with correct error when request body contains wrong data type (BOOLEAN)", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: false })
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toBe("Bad request");
+        });
+    });
```
Comment (Slyose) [inline app.test.js]: I would say that the second test is somewhat redundant as it's very similar to the first

**Review by Slyose — APPROVED** [review]
Good response to feedback, though I think one test will be sufficient as opposed to two. Feel free to merge solo after looking at comment!

**Pull request merged**